omnibus/files/private-chef-cookbooks/private-chef/files/default/html/index.html

```diff
@@ -10,20 +10,38 @@
 <body>
   <div class="header-block">
     <div id="header">
-      <strong class="logo"><a href="http://www.getchef.com">Chef</a></strong>
+      <strong class="logo"><a href="http://chef.io">Chef</a></strong>
     </div>
   </div>
   <div id="wrapper">
     <div id="main">
       <div class="mybox">
         <div id="content">
-          <h1>Chef Server API</h1>
+          <h1>Are You Looking For the Chef Server?</h1>
+
+          <p>Hello! It looks like you were trying to browse to your Chef Server but you haven't installed the Management Console.</p>
+
           <p>
-            This is the main endpoint for all of the Chef APIs. In general,
-            none of these have any HTML representations, and the vast majority
-            of them require that you are sending properly authenticated
-            requests. So while it's neat that you came to visit, you probably
-            won't find what you are looking for here.
+            If you prefer, you can access the server programmatically with the Chef Server API. To learn how to
+            do this, head over to our <a href="https://docs.chef.io/api_chef_server.html">API Documentation</a> pages.
+          </p>
+
+          <p>
+            If you do want to use your browser to manage your server, then
+            install the Management Console. It's free for up to 25 nodes.
+          </p>
+
+          <p>
+            One way to install the Management Console is to log in as root and type this command:
+            <pre> # chef-server-ctl install opscode-manage </pre>
```

Comment:
Note that this will be changed to `chef-manage` soon, but we'll make sure that happens when we make that migration.